boards/xtensa/intel_s1000_crb/board.cmake

```diff
@@ -1,2 +1 @@
-set(DEBUG_SCRIPT xt-debug.sh)
```

Comment:
Please combine these into a single commit. The older shell script-based way of working is not supported anymore.

Reply:
Done.